pkg/scheduler/cache/cache.go

```diff
@@ -85,6 +85,14 @@ type SchedulerCache struct {
 	schedulerName      string
 	nodeSelectorLabels map[string]string
 
+	// CustomInformers is an interface list for custom informers.
+	// It is the entry point for custom object event processing, which can be
+	// accessed to the scheduling framework by implementing the SharedInformer interface.
+	// These shared informer will start and run at the beginning of the scheduler.
+	// The CustomInformers will be used along with the EventHandler framework.
+	// Initialize the CustomInformers, while initializing the EventHandler object.
+	CustomInformers []cache.SharedIndexInformer
```

Comment:
How to add CustomerInformers and use it? is there a example or instruction for it in design doc or somewhere?

Reply:
please see test case in plugins_test.go
![image](https://user-images.githubusercontent.com/25945130/147380875-00cdc35a-4b8f-41f5-9659-f148d89dcd4b.png)